### PR TITLE
Add clean game command

### PR DESCRIPTION
**Pull request by jshemas**

### CODE REVIEW

**Comment by jshemas** [comment]
I wonder if I can use `ts-node` here. 🤔

**Comment by jshemas** [comment]
> I wonder if I can use `ts-node` here. 🤔

anything is possible

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ npm run compile:gameStateSchema
 npm run --silent dev
 ```
 
+Use `npm run clean:game` to delete old save files.
+
```
Comment (rachidlamouri) [inline README.md:16]: How about:
```diff
-```
-
-Use `npm run clean:game` to delete old save files.
+# Delete the current save file
+npm run clean:game
+```
```

File: package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "clean": "rimraf lib",
+    "clean:game": "ts-node -e 'const { databaseUtil } = require(\"./src/utils/databaseUtil.ts\"); databaseUtil.delete();'",
```
Comment (rachidlamouri) [inline package.json:6]: `ts-node` isn't doing anything differently here than `node`. I'd revert it back to `node`. I have a suggestion that would be easier to show in a separate PR

File: README.md (hunk 1 above)
Comment (jshemas) [reply README.md]: updated

File: package.json (hunk 2 above)
Comment (rachidlamouri) [reply package.json:6]: Actually why not just do `rm -r saves/`?

Comment (jshemas) [reply package.json:6]: > `ts-node` isn't doing anything differently here than `node`. I'd revert it back to `node`. I have a suggestion that would be easier to show in a follow up PR

Are you saying i should revert to `"clean:game": "node -e 'const { databaseUtil } = require(\"./src/utils/databaseUtil.ts\"); databaseUtil.delete();'",` or are you saying to revert to `"clean:game": "node -e 'const { databaseUtil } = require(\"./lib/utils/databaseUtil.js\"); databaseUtil.delete();'",`?

The first one will cause `SyntaxError: Cannot use import statement outside a module` and the second one can only be used if you run `npm run prepack`.

> Actually why not just do `rm -r saves/`?

The first version of this command was that but then I look at the task and it stated to call `delete`. I'm fine with whichever.

Comment (rachidlamouri) [reply package.json:6]: I like `rm -r saves`. It's simple.

Comment (jshemas) [reply package.json]: updated to use `rm`

**Review by rachidlamouri — APPROVED** [review]
